plugin_store/main.py

```diff
@@ -86,18 +86,28 @@ async def submit_plugin(self, request):
         version_name = data["version_name"]
         image_url = data["image"]
         file_bin = data["file"].file.read()
+        
+        if data["force"]:
+            force = bool(eval(data["force"].title()))
```

Comment:
what this line does is provide compatibility for all types of boolean inputs
* `data["force"].title()` makes false/true to become False/True, as used in python syntax
* `bool()` allows 0s and 1s to be used as False and True

Reply:
pls no eval :)

Reply:
eval gonzo :D